RELEASENOTES.md

```diff
@@ -1,5 +1,8 @@
 # Release notes
 
+## v2.4.8
+ - Reset state when calling videochange (#107)
```

Comment:
Just so we can remember better in the future (and so it makes updating the docs a bit better, where we want a little more user-friendly release notes), should we change this? I'm thinking something like:

 - Reset internal state when calling `videochange`, fixing an issue where rebuffering may be reported incorrectly after calling `videochange`

thoughts?

Reply:
Sounds good. I've just been using the PR title, but I agree that it makes sense to expand on the change a bit in the release notes.